drivers/hud_pit/spec/models/hud_pit/generators/pit/fy2025/pit_basic_counts_spec.rb

```diff
@@ -0,0 +1,427 @@
+        # Number of Persons (35 - 44) is cell B6 for the Adults question
+        age_35_44_count = report.answer(question: question, cell: 'B6')
+        expect(age_35_44_count.value).to eq(3)
+      end
+    end
+
+    context 'for Persons Ages 45-54' do
+      let(:question) { HudPit::Generators::Pit::Fy2025::Adults::QUESTION_NUMBER }
+      let(:dob_age_45) { pit_date - 45.years } # Exactly 45
+      let(:dob_age_50) { pit_date - 50.years } # Exactly 50
+      let(:dob_age_54) { pit_date - 54.years } # Exactly 54
+      let(:dob_age_44) { pit_date - 44.years } # Exactly 44 (too young)
+      let(:dob_age_55) { pit_date - 55.years } # Exactly 55 (too old)
+
+      before do
+        household_id = 'hh_age_45_54_1'
+        hoh_client = create_client_with_warehouse_link(uid: 'client_a4554_hoh', dob: pit_date - 60.years)
+        create_enrollment(client: hoh_client, project: es_project, entry_date: pit_date, relationship_to_ho_h: rel_hoh, household_id: household_id)
+
+        m1 = create_client_with_warehouse_link(uid: 'client_a4554_m1', dob: dob_age_45)
+        create_enrollment(client: m1, project: es_project, entry_date: pit_date, relationship_to_ho_h: rel_other_adult, household_id: household_id)
+        m2 = create_client_with_warehouse_link(uid: 'client_a4554_m2', dob: dob_age_50)
+        create_enrollment(client: m2, project: es_project, entry_date: pit_date, relationship_to_ho_h: rel_other_adult, household_id: household_id)
+        m3 = create_client_with_warehouse_link(uid: 'client_a4554_m3', dob: dob_age_54)
+        create_enrollment(client: m3, project: es_project, entry_date: pit_date, relationship_to_ho_h: rel_other_adult, household_id: household_id)
+
+        m_young = create_client_with_warehouse_link(uid: 'client_a4554_m_young', dob: dob_age_44)
+        create_enrollment(client: m_young, project: es_project, entry_date: pit_date, relationship_to_ho_h: rel_other_adult, household_id: household_id)
+        m_old = create_client_with_warehouse_link(uid: 'client_a4554_m_old', dob: dob_age_55)
+        create_enrollment(client: m_old, project: es_project, entry_date: pit_date, relationship_to_ho_h: rel_other_adult, household_id: household_id)
+      end
+
+      it 'counts only persons aged 45-54' do
+        report = run_report(questions: [question])
+        # Number of Persons (45 - 54) is cell B7 for the Adults question
+        age_45_54_count = report.answer(question: question, cell: 'B7')
+        expect(age_45_54_count.value).to eq(3)
+      end
+    end
+
+    context 'for Persons Ages 55-64' do
+      let(:question) { HudPit::Generators::Pit::Fy2025::Adults::QUESTION_NUMBER }
+      let(:dob_age_55) { pit_date - 55.years } # Exactly 55
+      let(:dob_age_60) { pit_date - 60.years } # Exactly 60
+      let(:dob_age_64) { pit_date - 64.years } # Exactly 64
+      let(:dob_age_54) { pit_date - 54.years } # Exactly 54 (too young)
+      let(:dob_age_65) { pit_date - 65.years } # Exactly 65 (too old)
```

Comment:
It feels like some of these age bucket tests could be DRYed up significantly by creating a hash of ages and looping over them.  No harm in the way it is written, it just feels like a LOT of code.